src/base-config/keyboard.json

```diff
@@ -293,7 +296,7 @@
             "platform": "mac"
         }
     ],
-    "navigate.nextDoc":  [
+    "next.recent.files":  [
```

Comment:
Also, normally extension keybindings are handled by the extension itself (see [RecentProjects](https://github.com/adobe/brackets/blob/54c646ae70bd92395bdc066c7ace3acf6f6c8e37/src/extensions/default/RecentProjects/keyboard.json), [2](https://github.com/adobe/brackets/blob/54c646ae70bd92395bdc066c7ace3acf6f6c8e37/src/extensions/default/RecentProjects/main.js#L449) as an example)